src/components/dialog.css

```diff
@@ -24,12 +24,52 @@
  * The maximum width allowed for the dialog is 480 px. Default width
  * is 100% (for container width < 480 px).
  */
-@media (min-width: 480px) {
+@media (max-width: 640px), (max-height: 640px) {
```

Comment:
Update the description above to reflect the new rules.

Reply:
Updated.